index.js

```diff
@@ -52,7 +52,11 @@ module.exports = {
     'no-unexpected-multiline': 'error',
     'no-unreachable': 'error',
     'no-unsafe-finally': 'error',
-    'no-unused-vars': ['error', {vars: 'all', args: 'none'}],
+    'no-unused-vars': ['error', {
+      vars: 'all',
+      args: 'after-used',
+      ignoreRestSiblings: true
+    }],
```

Comment:
It looks like Prettier wants this to be like:
```js
    'no-unused-vars': [
      'error',
      {
        vars: 'all',
        args: 'after-used',
        ignoreRestSiblings: true
      }
    ],
```

Looks like you could also just do `'no-unused-vars': ['error', {ignoreRestSiblings: true}]`.

Reply:
Sorry, I haven't install dependencies for development on this project. 
Your suggested value, is in fact, how it is set in one of our private repository. I will update it accordingly.